lib/concurrent/thread_safe/util/striped64.rb

```diff
@@ -83,7 +83,7 @@ class Cell < Concurrent::AtomicReference
 
           # TODO: this only adds padding after the :value slot, need to find a way to add padding before the slot
           # @!visibility private
-          attr_reader *(Array.new(12).map {|i| :"padding_#{i}"})
+          attr_reader *(12.times.collect{ |i| "padding_#{i}".to_sym })
```

Comment:
@thedarkone Can you please verify this change? When I load the gem under Ruby's verbose mode I was getting this output:

```
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
/home/jerry/Projects/ruby-concurrency/concurrent-ruby/lib/concurrent/thread_safe/util/striped64.rb:86: warning: method redefined; discarding old padding_
```

Here's what I get in IRB when I run the two different array operations:

```
2.2.3 :001 > 12.times.collect{ |i| "padding_#{i}".to_sym }
 => [:padding_0, :padding_1, :padding_2, :padding_3, :padding_4, :padding_5, :padding_6, :padding_7, :padding_8, :padding_9, :padding_10, :padding_11]
2.2.3 :002 > Array.new(12).map {|i| :"padding_#{i}"}
 => [:padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_]
```

Same in 1.9.3:

```
1.9.3-p551 :001 > 12.times.collect{ |i| "padding_#{i}".to_sym }
 => [:padding_0, :padding_1, :padding_2, :padding_3, :padding_4, :padding_5, :padding_6, :padding_7, :padding_8, :padding_9, :padding_10, :padding_11]
1.9.3-p551 :002 > Array.new(12).map {|i| :"padding_#{i}"}
 => [:padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_, :padding_]
```

I believe the updated code accomplishes what was intended here, but I don't want to merge this without your :+1: